### ISSUE TITLE
Other swarm nodes not seeing our content

### ISSUE THREAD

**Issue by jakubgs** [status-im/infra-ipfs#1]
Accordin to Barry we are not sharing our content with other gateways.
For example:
```
 $ curl -s https://test-ipfs.status.im/ipfs/QmZdER7pR3S4NCri89kf5qVEZfDWvFSwLEYyAWRWdwQgHt    
sup my dude
 $ curl -s --max-time 10 https://ipfs.infura.io/ipfs/QmZdER7pR3S4NCri89kf5qVEZfDWvFSwLEYyAWRWdwQgHt       
 $ echo $?
28
```
Which appears to be true.

**Comment by jakubgs** [comment]
If I check the connected swarm nodes there's plenty:
```bash
 $ curl -s http://localhost:5001/api/v0/swarm/addrs | jq . | head -n10                                                                                                     
{
  "Addrs": {
    "12D3KooWA18Vv4VAPFfyeV8f9kmEuP23wdTGesZsW95er82XEN19": [
      "/ip4/172.31.3.58/tcp/20072",
      "/ip4/52.56.144.76/tcp/20072"
    ],
    "12D3KooWALLSYv1tvvVdSLq6MJJ7LZKaPKSLBGUHoVDuxHZHJWPv": [
      "/ip4/172.31.34.209/tcp/20046",
      "/ip4/54.166.205.235/tcp/20046"
    ],
```
```bash
 $ curl -s http://localhost:5001/api/v0/swarm/addrs | jq '.Addrs | keys | length' 
669
```

**Comment by jakubgs** [comment]
And we have plenty of peers:
```bash
 $ curl -s http://localhost:5001/api/v0/swarm/peers  | jq -r '.Peers[].Peer' | wc -l
82
```

**Comment by jakubgs** [comment]
Relevant and pretty fresh discuss post:
https://discuss.ipfs.io/t/cannot-make-any-file-available-to-the-ipfs-network/6014/2

Where `chris` states that:
>TLDR: For the past few months content routing (Who can I acquire this data from?) has been having issues, fixes are in the works.

Possibly related issue based on the post:
https://github.com/libp2p/go-libp2p-kad-dht/issues/345

**Comment by jakubgs** [comment]
A new member joined Status recently - @dryajov - which according to HR email has experience with IPFS. Maybe he can help me out with this, so I sent him an email.
